### PR TITLE
Add more question

### PR DESCRIPTION
**Pull request by louvelmathieu**
More question to train for the certification.

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
Hello Mathieu, 

I'll take a look asap, thanks for your help and good luck for the test ;)

Mickaël

File: data/bundles.yml [hunk 1]
```diff
@@ -68,3 +68,12 @@ questions:
         answers:
             - {value: "True",    correct: true}
             - {value: "False",   correct: false}
+    -
+        question: 'According to Symfony best practices, which files files are mandatory in your bundle AcmeBlogBundle directory Structure ?'
+        answers:
+            - {value: "AcmeNameBundle.php", correct: true}
```
Comment (20uf) [inline data/bundles.yml]: Wrong or `AcmeBlogBundle.php`

**Comment by 20uf** [comment]
Hello,

This is just my opinion, but I think it would be better if the information in "help" is added, it also allows to check and control the new additions.

```
        question: 'Symfony is released under which license ?'
        answers:
            - {value: "GNU General Public License (GPL)", correct: false}
            - {value: "ISC License", correct: false}
            - {value: "BSD license", correct: false}
            - {value: "MIT license", correct: true}
        help: |
            http://symfony.com/doc/current/contributing/code/license.html
```
It does not bother me to go back to the old questions to add this information.

File: data/bundles.yml (hunk 1 above)
Comment (louvelmathieu) [reply data/bundles.yml]: It's fix

**Comment by louvelmathieu** [comment]
The help option is a good idea but it should be had to https://github.com/certificationy/certificationy-cli with a new option.

**Comment by 20uf** [comment]
Aids are displayed only when the results are displayed, see:

image

**Comment by 20uf** [comment]
Thanks @louvelmathieu

**Pull request merged**